internal/service/streamprivatelinkendpoint/resource_schema.go

```diff
@@ -0,0 +1,69 @@
+package streamprivatelinkendpoint
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func ResourceSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed:            true,
+				MarkdownDescription: "The ID of the Private Link connection.",
+			},
+			"dns_domain": schema.StringAttribute{
+				Required:            true,
+				MarkdownDescription: "Domain name of Privatelink connected cluster.",
+			},
+			"dns_sub_domain": schema.ListAttribute{
+				Optional:            true,
+				MarkdownDescription: "Sub-Domain name of Confluent cluster. These are typically your availability zones.",
+				ElementType:         types.StringType,
```

Comment:
curious is this an actual array in the API? Just want to check that the Open API is correct.

Reply:
Yes it is an array, see the [API docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Streams/operation/createPrivateLinkConnection)